main/ui/src/main/java/org/cryptomator/ui/preferences/GeneralPreferencesController.java

```diff
@@ -171,4 +209,17 @@ protected Void call() throws Exception {
 		}
 	}
 
+	private KeychainBackend[] getAvailableBackends() {
+		if (!keychain.isPresent()) {
+			return new KeychainBackend[]{};
+		}
+		if (SystemUtils.IS_OS_LINUX) {
+			EnumSet<KeychainBackend> backends = LinuxSystemKeychainAccess.getAvailableKeychainBackends();
+			return backends.size() > 0 ? backends.toArray(KeychainBackend[]::new) : new KeychainBackend[]{};
```

Comment:
I commented this in #1367 already. This shouldn't be necessary:

```suggestion
			return backends.toArray(KeychainBackend[]::new);
```

Reply:
I checked that. You are right, it's not necessary and I removed it.